### PR TITLE
INVGOV-9839 add status endpoint

### PR DESCRIPTION
**Pull request by ParryLuke**
Is it okay to allow this call anonymously? Amazon can't send an API key header to authenticate the request for the health check.

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -9,6 +9,23 @@ security:
...
+          description: Status returned successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    description: Status of the API
+                    type: string
+                    example: 'Ok'
```
Comment (jmorahan) [inline api.swagger.yaml]: If we are returning a string we should either document the possible values and what they mean, or if it's an unpredictable message intended to be read by humans, at least document that fact explicitly.

File: api.swagger.yaml [hunk 2]
```diff
@@ -9,6 +9,23 @@ security:
...
+      description: Returns the status of the Invotra API. If successful, the API is running as expected.
```
Comment (jmorahan) [inline api.swagger.yaml]:
```suggestion
      description: Returns the status of the Invotra API. If successful, the API is running as expected. (If it is not, an appropriate 5xx error message will be returned).
```

While we don't explicitly document 5xx errors, I think it's worth adding an explanation of why this endpoint doesn't have a negative response.

**Comment by jmorahan** [comment]
I don't see a problem with allowing anonymous access to this endpoint btw

**Comment by sam-ludlow** [comment]
Please see the task INVGOV-9839 for comment about using the datetime rather than "OK" in the response.

File: api.swagger.yaml [hunk 3]
```diff
@@ -2629,6 +2641,20 @@ components:
           type: string
           example: '#FFFFFF'
       type: object
+    status-schema:
+      properties:
+        status:
+          description: 'Status of the API.'
+          type: string
+          example: 'OK'
```
[the remark this answers is not in the document]
Comment (sam-ludlow) [reply api.swagger.yaml]: done

File: api.swagger.yaml [hunk 4]
```diff
@@ -2629,6 +2641,20 @@ components:
...
+        timestamp:
+          description: 'Time on the server as a timestamp (in seconds).'
+          type: integer
+          example: 1563357496
+        datetime:
+          description: 'Time on the server in a human readable format.'
```
[the remark this answers is not in the document]
Comment (sam-ludlow) [reply api.swagger.yaml]: done

**Pull request merged**